### DISCUSSION TITLE
Release 0.1.5?

### DISCUSSION THREAD

**Discussion by tsmith123** [ACINQ/phoenixd#48] [General]
Is there going to be a release of 0.1.5 any time soon? I think there's a few things to go in it.

**Comment by tsmith123** [comment]
There was activity from the maintainers on this repo pretty much everyday until the Samurai news a few weeks back. Now, all has gone quiet. An update would be good please Acinq devs 🙏

**Comment by pm47** [comment]
phoenixd is a thin wrapper around [lightning-kmp](https://github.com/ACINQ/lightning-kmp), where most of the dev is happening. We are currently focused on bolt 12 support.

Will probably do a release after #50 is merged.

**Reply by tsmith123** [reply]
Appreciate the update. Many thanks

**Comment by pm47** [comment]
https://github.com/ACINQ/phoenixd/releases/tag/v0.1.5
